test/integration/BUILD.bazel

```diff
@@ -76,6 +78,20 @@ java_gapic_test(
     runtime_deps = ["@com_google_googleapis//google/cloud/asset/v1:asset_java_gapic_test"],
 )
 
+# Showcase API.
+java_gapic_test(
```

Comment:
Do we need this bazel target?

Reply:
This depends on how we want to invoke the showcase tests. Either they are to be integrated with the existing `//test/integration` stages, or we create a separate workflow to test them. I had assumed the `integrated with the existing` route due to how similar the syntax is (so far). Once we start working with the server, the tests and syntax will quickly diverge.

How would you like to handle this?

Reply:
I think either way makes sense, we are not moving away from the golden integration tests yet soon, so we can continue to test the golden files and the generated unit tests in integration folder, while all the future handwritten showcase tests will be in the showcase folder. Or we move all showcase related stuff to showcase folder.

Reply:
Kept here, and removed from /test/showcase.